src/client/service/Menu.java

```diff
@@ -0,0 +1,65 @@
+package client.service;
+
+import java.io.IOException;
+import client.Client;
+import client.model.MenuItem;
+import client.requestgateway.MenuRequestGateway;
+import client.util.PrintOutToConsole;
+import client.util.MenuItemHandler;
+import client.util.RequestHandler;
+
+public class Menu {
```

Comment:
Class name is menu but it acts as a helper class for menu related operations. Please rename accordingly.